src/main/java/com/itacademy/cms/model/User.java

```diff
@@ -0,0 +1,97 @@
+package com.itacademy.cms.model;
+
+import com.itacademy.cms.model.enums.Role;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @NotNull
+    private String name;
+
+    @NotNull
+    private String surname;
+
+    @NotNull
+    private String email;
+
+    @NotNull
+    private String password;
+
+    private double accountCard;
+
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    private Role role;
+
+    private String about;
+
+    public String getName() {
```

Comment:
so just use `@Getter @Setter` too I guess

Reply:
I'll take it, thanks